src/ismn/filehandlers.py

```diff
@@ -22,6 +22,9 @@
 
 import os
 import pandas as pd
+import warnings
+
+warnings.simplefilter(action="ignore", category=UserWarning)
```

Comment:
This catches all warning no? Is this intentional? generally warning are useful

Reply:
Was a leftover from debugging, removed it now